docs/quickstart.md

```diff
@@ -38,6 +38,8 @@ two Trino servers and a PostgreSQL database running in Docker, and a Trino
 Gateway server running in the host operating system. If you are using a Mac and
 have the `brew` package manager installed, this script attempts to install
 `psql`, if not you must install `psql` through other means.
+- If you use `brew` package, you might have to set libpq first in your PATH, run:
+- `echo 'export PATH="/opt/homebrew/opt/libpq/bin:$PATH"' >> ~/.zshrc`
```

Comment:
This assume usage of zsh ... dont do that. Just say what folder has to be added to the PATH. Something like 

Use the following command in your shell startup script or manually: 

```
export PATH="/opt/homebrew/opt/libpq/bin:$PATH"
```